deployment/terraform/main.tf

```diff
@@ -0,0 +1,356 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "<= 5.72.1"
+    }
+  }
+}
+
+provider "aws" {
+  region                     = "us-east-1"
+  skip_requesting_account_id = var.local_run
+}
+
+variable "local_run" {
+  description = "Flag to determine if running locally using LocalStack"
+  type        = bool
+}
```

Comment:
It is not necessary, in my code I wanted to keep the same concepts you already had in the repository. I can get rid of the `local_run` variable.  

I can certainly get rid of the `local_run` variable. I think there are two main ways to address this:
1. propagate endpoints as env variables to python code
2. use [transparent endpoint injection](https://docs.localstack.cloud/user-guide/tools/transparent-endpoint-injection/). Will this approach work fine in CI/CD pipelines?

Reply:
> So it's basically really just about setting the STAGE environment variable for the Lambdas to local, right?
yes, exactly

> If you want to give it a try, I think the transparent endpoint injection should work just fine, however modifying the Lambda code is out of scope of this PR in my opinion. ;)

> So I think we could really just remove the terraform variable, set the STAGE environment variable for all lambdas (since the deployments are not tested against AWS yet anyways as you outlined and analyzed in your comment below), and just add a TODO comment explaining why this is necessary currently. :)

ok 👍